ocs_ci/ocs/cluster.py

```diff
@@ -500,6 +500,80 @@ def enable_health_monitor(self, sleep=5):
     def disable_health_monitor(self):
         self.health_monitor_enabled = False
 
+    def get_ceph_cluster_iops(self):
+        """
+        The function gets the IOPS from the ocs cluster
+
+        Returns:
+            Total IOPS in the cluster
+        """
+
+        ceph_status = self.get_ceph_status()
+        for item in ceph_status.split("\n"):
+            if 'client' in item:
+                iops = re.findall(r'\d+\.+\d+|\d\d*', item.strip())
+                read_iops = iops[2]
+                write_iops = iops[3]
+                iops_in_cluster = float(read_iops) + float(write_iops)
+                return iops_in_cluster
+
+    def get_iops_percentage(self, osd_size=2):
+        """
+        The function calculates the IOPS percentage
+        of the cluster depending on number of osds in the cluster
+
+        Args:
+            osd_size (int): Size of 1 OSD in Ti
+
+        Returns:
+            IOPS percentage of the OCS cluster
+        """
+
+        osd_count = count_cluster_osd()
+        iops_per_osd = osd_size * constants.IOPS_FOR_1TiB_OSD
+        iops_in_cluster = self.get_ceph_cluster_iops()
+        osd_iops_limit = iops_per_osd * osd_count
+        iops_percentage = (iops_in_cluster / osd_iops_limit) * 100
+        assert iops_in_cluster <= osd_iops_limit, f"The IOPS in the cluster" \
```

Comment:
Why we need to assert here ?, we might see in some cases where it goes beyond the limit